lib/rules/comma-dangle.js

```diff
@@ -12,7 +12,9 @@
 //------------------------------------------------------------------------------
 
 module.exports = function (context) {
-    var forbidDanglingComma = context.options[0] !== "always";
+    var allowDangle = context.options[0] || "never";
```

Comment:
@nzakas is it so? Looks like the default (if the option argument is undefined) is still `never`.